schema/filter_artworks.js

```diff
@@ -186,6 +216,7 @@ function filterArtworks(primaryKey) {
       }
 
       return gravity.with(accessToken)("filter/artworks", gravityOptions)
+        .then(response => assign({}, response, { options: gravityOptions }))
```

Comment:
I guess I still don't fully understand where we'll use this. Returning the tag I can sort of see- we're filtering by the tag and at a minimum we'll need some sort of 'name' for the thing we're filtering, so that makes sense.

But these other filter options I'm still confused about. If were filtering by a tag, and also filtering by price, 'for sale', and some medium- my naive thinking would assume those exist as state (or props maybe) on a component, and not need to be included in a filter response.